iot_control/models/iot_partner_group.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class IotPartnerGroup(models.Model):
+    _name = "iot.partner.group"
```

Comment:
Same as `iot.partner`, i think it might be better `iot.control.security.group`.

Also, I don't get de difference between this table and iot.control. Shouldn't they be the same?

Reply:
iot.control defines rules.

A rule binds
Groups of Persons  (iot.partner.group)
with 
Groups of Devices (iot.device.group).